vulnerabilities/models.py

```diff
@@ -67,6 +70,12 @@ def paginated(self, per_page=5000):
 
 
 class VulnerabilityQuerySet(BaseQuerySet):
+    def affecting_vulnerabilities(self):
```

Comment:
Please add tests for this in `test_models.py`

Reply:
@TG1999 Tests added yesterday for all property and other methods added as part of this PR.  

`make test`:  `325 passed, 1 skipped, 1 deselected, 9 warnings`